content/concept-authorized-trust-lines.md

```diff
@@ -0,0 +1,99 @@
+# Authorized Trust Lines
+
+The XRP Ledger's Authorized Trust Lines feature enables a currency issuer to limit who can hold its issued currencies, so that unknown XRP Ledger addresses cannot hold those currencies. The Authorized Trust Lines feature has no effect on XRP.
```

Comment:
Nit for your consideration: It took a little more reading to understand the implications of this last sentence above. This excerpt from content/concept-depositauth.md made it clear for me: "The [Authorized Trust Lines](concept-authorized-trust-lines.html) feature (`RequireAuth` flag) limits which counterparties can hold non-XRP currencies issued by an account."

I wonder if stating "non-XRP currencies" somewhere here would help?

> The Authorized Trust Lines feature has no effect on XRP and only limits which counterparties can hold **non-XRP** currencies issued by an account.